lambda/query-activity-log/query-activity-log.ts

```diff
@@ -60,9 +60,7 @@ export const validateTxmaEventBody = (txmaEvent: TxmaEvent): void => {
   ) {
     validateUser(txmaEvent.user);
   } else {
-    throw new Error(
-      `Could not validate UserServices ${JSON.stringify(txmaEvent)}`
```

Comment:
non PII data from the TXMA event could still be helpful to triage issues. eg the client_id, event_name.
Could this be logged?

Reply:
So these messages go to the DLQ and we log the message ID there. We can get the entire paylod if we look at the message in the DLQ ? WOuld it be helpful to still log client_id and event_name

Reply:
Ah sorry we don't log message Id for messages sent to DLQ maybe that would be a better option ?

Reply:
yeah, message ID seems key.  If the error log gives us message ID and we can see the message content from DLQ then that's grand